### PR TITLE
docs: new VScode 1.14 default protocol changed.

### LINKED ISSUE #1206: vscode升级1.14后默认protocol由 Legacy 变为了 Inspector
* **Node Version**: 6.3.1
* **Egg Version**: 3.0.4
* **Plugin Name**:
* **Plugin Version**:
* **Platform**: OSX
* **Mini Showcase Repository**:

vscode升级1.14后，默认protocol由 Legacy 变为了 Inspector,  如果原来用的是 Legacy， 升级后vscode调试会跑不起来。

需要修改文档：https://eggjs.org/zh-cn/core/development.html#使用-vscode-进行调试 ，
在 .vscode/launch.json 的Launch Egg字段添加 
```
"protocol": "legacy"
```

详情参考：https://github.com/Microsoft/vscode/issues/30629#issuecomment-315196813

### PR DESCRIPTION
**Pull request by anto1717**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

##### Description of change
change docs:  [https://eggjs.org/zh-cn/core/development.html#使用-vscode-进行调试](https://eggjs.org/zh-cn/core/development.html#使用-vscode-进行调试)

references: [https://github.com/Microsoft/vscode/issues/30629#issuecomment-315196813](https://github.com/Microsoft/vscode/issues/30629#issuecomment-315196813)

Closes #1206

### CODE REVIEW

File: docs/source/zh-cn/core/development.md [hunk 1]
```diff
@@ -389,4 +392,4 @@ exports.proxyworker = {
 [egg-bin]: https://github.com/eggjs/egg-bin
 [VSCode]: https://code.visualstudio.com
 [Legacy Protocol]: https://github.com/buggerjs/bugger-v8-client/blob/master/PROTOCOL.md
-[Inspector Protocol]: https://chromedevtools.github.io/debugger-protocol-viewer/v8
\ No newline at end of file
+[Inspector Protocol]: https://chromedevtools.github.io/debugger-protocol-viewer/v8
```
Comment (iyuq) [inline docs/source/zh-cn/core/development.md]: 最后一行不是没变化么

File: docs/source/zh-cn/core/development.md [hunk 2]
```diff
@@ -366,8 +367,10 @@ exports.proxyworker = {
 
 当且仅当你的 Node.js 版本大于 7.x 时，可以使用 [Inspector Protocol] 进行调试。
 
+VScode升级1.14后，默认protocol由 Legacy 变为了 Inspector，如果仍需使用Legacy, 要手动指定。详情参考 https://github.com/Microsoft/vscode/issues/30629#issuecomment-315196813
```
Comment (popomore) [inline docs/source/zh-cn/core/development.md]: 中英文之间需要加空格

Comment (popomore) [inline docs/source/zh-cn/core/development.md]: 详情参考 [issue](https://github.com/Microsoft/vscode/issues/30629#issuecomment-315196813)

**Comment by anto1717** [comment]
已修复：中英文之间添加空格。

**Pull request merged**